src/execFile.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * LiskHQ/lisky
+ * Copyright © 2017 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import fs from 'fs';
+import childProcess from 'child_process';
+
+const handleError = (lisky, error) => lisky.log(error.trim ? error.trim() : error) || 'XXX';
```

Comment:
The `||` operator seems unnecessary here

Reply:
Oh whoops! Should have been removed, sorry.